CONTRIBUTING.md

```diff
@@ -148,6 +148,9 @@ the code. It also lets the community know what you're working on, and if you
 need help, you can reference the issue when discussing it with other community
 and team members.
 
+* For documentation issues relating to pages in the Start, Build, Transpile, Verify, Run, and Migration guides sections of docs.quantum.ibm.com, please open an issue in the [Qiskit/documentation repo](https://github.com/Qiskit/documentation/issues/new/choose) rather than the Qiskit/qiskit repo. In other words, any page that DOES NOT have `/api/` in the url should be addressed in the Qiskit/documentation repo. (Exception: the Migration guide urls contain `/api/` but are managed in the Qiskit/documentation repo.)
```

Comment:
I think it could be helpful to make the reference to docs.quantum.ibm.com a link. I also thought about adding a link to the migration guides section (https://docs.quantum.ibm.com/api/migration-guides) just because it's a bit trickier to find, but didn't want to overdo it with the links. What do you think?

```suggestion
* For documentation issues relating to pages in the Start, Build, Transpile, Verify, Run, and Migration guides sections of [docs.quantum.ibm.com](https://docs.quantum.ibm.com/), please open an issue in the [Qiskit/documentation repo](https://github.com/Qiskit/documentation/issues/new/choose) rather than the Qiskit/qiskit repo. In other words, any page that DOES NOT have `/api/` in the url should be addressed in the Qiskit/documentation repo. (Exception: the Migration guide urls contain `/api/` but are managed in the Qiskit/documentation repo.)
```

Reply:
I can't comment on the correct line (138), but I would also change the title of the section from "Pull requests" to "Issues and pull requests", so that the issue-related guidelines are easier to find/

Reply:
Thanks @ElePT , great suggestions. I'll make the migration guides link and update the header.